libs/docs/src/docs/no-version/quickstart-odo.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "Quick start with odo v3"
+description: "Quick start with odo v3"
+---
+
+This guide will run through creating a simple hello world devfile project using `odo` version 3. The purpose of this guide is to provide a practical introduction to those just starting out with devfiles.
+
+## Prerequisites
+
+- `odo` v3
+- `kubectl` or `oc`
+- Docker 17.05 or higher
+- Access to a Kubernetes or OpenShift cluster
+
+## Procedure
+
+1. Access or setup your target cluster
+    - (Optional) You can use `minikube` to run your cluster locally, follow [these](https://minikube.sigs.k8s.io/docs/start/) steps to get started
+
+2. Install `odo` version 3 if you do not already have it by following the [Installation guide](https://odo.dev/docs/overview/installation)
+
+3. Create a directory to store a simple [hello world Express.js](https://expressjs.com/en/starter/hello-world.html) application
+    
+    ```json {% title="package.json file" filename="package.json" %}
+    {
+        "name": "helloworld-devfile",
+        "version": "1.0.0",
+        "description": "",
+        "main": "app.js",
+        "type": "module",
+        "dependencies": {
+            "@types/express": "^4.17.17",
+            "express": "^4.18.2"
+        }
+    }
+    ```
+
+    ```js {% title="Application source code" filename="app.js" %}
+    import express from "express"
+
+    const PORT = 3000
+    const app = express()
+    
+    app.get("/", (req, res) => {
+        res.send("Hello world!")
+    })
+
+    app.listen(PORT, () => {
+        console.log(`Listening on port ${PORT}..`)
+    })
+    ```
+
+4. Create a devfile with the filename `.devfile.yaml`. Add the [`schemaVersion`](./devfile-schema#schema-version) field with the desired devfile specification version to use
+
+    ```yaml {% filename=".devfile.yaml" %}
+    schemaVersion: <version>
```

Comment:
if we are showing snippets of the devfile content, it might be better to make it easier for users and include the actually version

Reply:
> if we are showing snippets of the devfile content, it might be better to make it easier for users and include the actually version

It will include the actual schema version, `<version>` is a placeholder for printing the selected schema version dynamically in that spot. So if you load the site under the `2.2.0` doc pages, `<version>` will be replaced by `2.2.0`.

Reply:
oh, did not know that. That is really smart 👍